index.html

```diff
@@ -374,5 +374,28 @@ <h2 class="top3-1-p">LET’S CONNECT</h2>
             </div>
         </div>
     </section>
+    <!-- the contact form's section -->
+    <section class="form">
+        <div class="contact-me">
+            <h2 class="form-h2"><span>Contact me</span></h2>
+            <p class="txt-p">
+                <span>
+                    If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+                </span>
+            </p>
+        </div>
+    
+      <form class="form-1" action="https://formspree.io/f/mzbowdel" method="post">
+        <ul>
+            <li><input type="text" class="name"  maxlength="30" placeholder="Yeremias" name="name" required></li>
+            <li><input type="email" class="email"  placeholder="natajayanj14@gmail.com"  name="email" required></li>
+            <li><textarea class="msg"  maxlength="500" placeholder="Write your message here" name="message" required></textarea></li>
+            <li><button type="submit" class="btn1">Get in touch</button></li>
+        </ul> 
+      </form>
```

Comment:
- [ ] **[OPTIONAL]:** You did a great job working on this milestone! 💯 To make your project even better, I would suggest adding some padding to each input field, this way, the text that users add doesn't look like is touching the edges! 🤓 🔧 

![image](https://user-images.githubusercontent.com/77069011/220448262-de2aabd9-be30-4346-800d-004092313b4f.png)